cmd/agent/agent.go

```diff
@@ -0,0 +1,32 @@
+package agent
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+
```

Comment:
unused packages will make the go compiler fail! ;)

Reply:
Thanks, fixed!